api/justfile

```
@@ -40,6 +40,9 @@ export PGCLI_VERSION := `grep -A 1 'name = "pgcli"' pdm.lock | grep 'version' |
 install *args:
     pdm install {{ args }}
 
+add *args:
+    pdm add {{ args }}
```

Maybe just a general `pdm` recipe, if this is really necessary? Why not run `ov pdm add` from within the api directory, though?

```suggestion
pdm *args:
    pdm {{ args }}
```

Adding this in just the API justfile is kind of tricky though, it makes everything else that uses PDM inconsistent, right?

You can pass `-p <project directory>` to pdm from the repo root to target a project. I do this often: `ov pdm -p api [...whatever]`. A root justfile recipe `pdm` that requires a target project would be more consistent and work for everything using PDM without creating inconsistencies of recipe availability in different projects.

---

Thank you for the `-p` suggestion! I actually added this as a quick workaround and forgot to remove it before pushing the PR